### ISSUE TITLE
Syntax Error in AI-Generated Code Snippet with TypeScript (TSX) in Code Sandbox

### ISSUE THREAD

**Issue by Yash-Patidar** [Nutlope/llamacoder#1]
**When generating a code snippet using the AI model, the code is rendered with an incorrect syntax that causes a syntax error in a TypeScript (TSX) file. The generated code snippet includes "```tsx" at the beginning, which is not a valid TypeScript syntax and results in an error when executed.**

**prompt** : build the chat APP
**AI model** : Gemma 2 27B
![Screenshot from 2024-08-03 02-22-49](https://github.com/user-attachments/assets/efb0d7ba-5f40-4df7-90d7-e9aad7ba7223)

**Comment by syeddhasnainn** [comment]
did you try again? working fine for me on all of the models

**Comment by Nutlope** [comment]
I tried to optimize the prompt so this shouldn't happen! Lemme know if it's still happening

**Comment by Yash-Patidar** [comment]
![Screenshot from 2024-08-03 12-20-58](https://github.com/user-attachments/assets/232dd676-6ca2-404c-a569-3f4fb8c2eef0)
Yes, I am still facing the same issue. However, sometimes it works well when we try again.

**Comment by kamaravichow** [comment]
Try removing such tags with regex 
```js
          text = text.replace(/```(tsx|typescript|javascript)?/g, "");
```

**Comment by yavin5** [comment]
Yes, it's still happening.  Maybe infrequently, but it is.  The TS code can filter those out, as @kamaravichow said above.

**Comment by Nutlope** [comment]
Sounds good, thanks for letting me know! I reviewed @kamaravichow's PR and requested some changes, so we'll try to get this fixed :)

**Comment by jachenms** [comment]
![image](https://github.com/user-attachments/assets/55d543ef-6289-4c0a-aab9-58a8d3c86fe0)
how to solve the issue when run my prompt
